lib/watir/locators/element/selector_builder/xpath.rb

```diff
@@ -49,7 +49,8 @@ def attribute_expression(building, selectors)
           def equal_pair(building, key, value)
             if key == :class
               if value.strip.include?(' ')
-                Watir.logger.deprecate "Using the :class locator to locate multiple classes with a String value", "Array"
+                Watir.logger.deprecate "Using the :class locator to locate multiple classes with a String value (i.e. #{value})",
+                                       "Array (e.g. [#{value.gsub(' ', ', ')}])"
```

Comment:
Like the idea, but I don't think the Array comes out right. A value of "a b" will create the message "Array (e.g. [a, b])". We should include the quotes around each value. 

I think you want `"Array (e.g. #{value.split})"` to give "Array (e.g. ["a", "b"])".

Reply:
this was a dumb mistake, thanks